tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -84,6 +86,8 @@
     "col_indices": "self",
     "ccol_indices": "self",
     "row_indices": "self",
+    "_from_buffer": "self",
```

Comment:
There are views with metadata change. They should be moved to the list above (also .values but this is beyond the scope of this PR)